Python/Lessons1/HW1Python.py

```diff
@@ -0,0 +1,62 @@
+##Задача 2: Найдите сумму цифр трехзначного числа.
+##*Пример:*
+##123 -> 6 (1 + 2 + 3)
+##100 -> 1 (1 + 0 + 0) |
+"""
+n = input("Введите трехзначное число: ")
+n = int(n)
+ 
+d1 = n % 10
+d2 = n % 100 // 10
+d3 = n // 100
+
+print("Сумма цифр числа:", d1 + d2 + d3)
+"""
+
+
+##Задача 4: Петя, Катя и Сережа делают из бумаги журавликов. Вместе они сделали S журавликов. Сколько журавликов сделал каждый ребенок, 
+# если известно, что Петя и Сережа сделали одинаковое количество журавликов, а Катя сделала в два раза больше журавликов, чем Петя и Сережа вместе?
+##*Пример:*
+##6 -> 1  4  1
+##24 -> 4  16  4
+##60 -> 10  40  10
+"""
+a = int(input())
+k = int((a/3)*2)
+p = int((k/2)/2)
+s = int(p)
```

Comment:
Можно использовать целочисленное деление.
```suggestion
k = s = a // 3 * 2
p = k // 2 // 2
```